### ISSUE TITLE
Clank will need to support (and install) 9.4 postgresql

### ISSUE THREAD

**Issue by steve-gregory** [iPlantCollaborativeOpenSource/clank#83]
https://www.postgresql.org/download/linux/ubuntu/ for Ubuntu 14.04 and older versions

**Comment by steve-gregory** [comment]
Clank should also uninstall 9.3 _or_ explicitly stop all postgresql services and then explicitly start the *selected* version of postgresql to ensure atmosphere app does not fail to make migrations dependent on postgresql >9.3

**Comment by amercer1** [comment]
Two edge cases right? 
If clank is running on ubuntu 14.04
1) install postgresql 9.4 if fresh install 
2) shutdown 9.3, install 9.4 and upgrade cluster(s)

https://medium.com/@tk512/upgrading-postgresql-from-9-3-to-9-4-on-ubuntu-14-04-lts-2b4ddcd26535#.9t9fkpl4u

**Comment by lenards** [comment]
I think we want to look at going from 9.3 to 9.5.

**Comment by lenards** [comment]
Note - we do not use pg in a clustered scenario ... our upgrade / migration steps will be slightly different:
- https://www.postgresql.org/docs/current/static/release-9-4.html
- https://www.postgresql.org/docs/current/static/release-9-5.html

**Comment by lenards** [comment]
In the spirit of _reuse_, I have found an example of [updating from pg 9.1 to 9.4](https://github.com/dpla/automation/blob/1e9f568490969c8b63c1acb139802bc1c6c2e38a/ansible/playbooks/migrations/upgrade-postgresql-201510.yml#L56).

**Comment by lenards** [comment]
#92 addresses just including the PostgreSQL Apt repository. 

We still need to consider automating the upgrade portion to consider this _resolved_.

**Comment by lenards** [comment]
So, I have been out of the Postgres _game_ for a couple of years. I was confused about `pg_upgrade` versus `pg_upgradecluster`. I didn't find too much info about the difference between the two, just that most _"guides"_ favor `pg_upgradecluster` (like the one linked above). 

So, `pg_upgradecluster` is one of several _cluster_ tools included in the `postgresql-common` packages that we are installing in all deployed environments. 

Listing the cluster is one thing you get as well:
```
vagrant@vagrant-ubuntu-trusty-64:~$ pg_lsclusters 
Ver Cluster Port Status Owner    Data directory               Log file
9.3 main    5432 online postgres /var/lib/postgresql/9.3/main /var/log/postgresql/postgresql-9.3-main.log
```

So - whatever is done in Ansible, using `pg_upgradecluster` is the path of least resistance!

**Comment by lenards** [comment]
Clank now supports upgrading to PostgreSQL 9.5 (or a target `pg_version`).

**Comment by lenards** [comment]
Considered fixed by #98.

**Comment by lenards** [comment]
Note, `postgresql-common` is a Debian package: https://github.com/jbq/postgresql-common
